data/json/items/ammo.json

```diff
@@ -387,7 +387,7 @@
         "name" : "hydrogen canister",
         "symbol" : "=",
         "color" : "light_gray",
-        "description" : "A canister of pressurized hydrogen so heavily compressed it has turned solid.  With proper equipment, it could be heated to plasma.",
+        "description" : "A canister of pressurized hydrogen.  With proper equipment, it could be heated into plasma.",
```

Comment:
We were discussing this in discord, there's no scientific possibility of carrying around solid hydrogen in a steel canister in the next 30 years. You'd need pressure and heat equivalent to that in the interior of Jupiter. Kevin is removing CBM and mininuke recipes for the sake of "realism"; guns using solid hydrogen is so far beyond that. It makes much more sense for it to be a gas, or even a liquid. I'm planning on changing the descriptions for the Boeing XM-P to gas as well for consistency.

Reply:
The problem is when some player wondering why they cant make this stuff themselves later, which is why this part was added in the first place. If this get removed, at least add a statement to point out that you cant make one yourself.

Reply:
Thinking over it, I anticipate that this change will be a blocker for the rest of the copyedits going in. I'm going to revert this back to the original and open a separate PR for it, where it can receive due discussion.